compliance/__init__.py

```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 """Compliance automation package."""
 
-__version__ = '1.2.5'
+__version__ = '1.2.6'
```

Comment:
- An automatic release is not performed upon merge for this repo.  So I'd remove the version bump and we'll handle it when we make the release (unless the plan is to make the release immediately).
- We are however keeping a CHANGES.md so you'll need to make an entry there.  If the release will be performed later (the likely case) then add an `# UNRELEASED` section to the top of CHANGES.md and then add what's been changed.
- If the plan is to make the release immediately after this is merged then I'd bump the version to 1.3.0 because this seems like new functionality.